src/test/java/com/transferwise/openbanking/client/jwt/JwtClaimsSignerTest.java

```diff
@@ -24,20 +19,15 @@
 import org.junit.jupiter.api.Test;
 import org.mockito.Mockito;
 
-import java.math.BigInteger;
 import java.security.KeyPair;
-import java.security.KeyPairGenerator;
 import java.security.cert.X509Certificate;
-import java.time.LocalDateTime;
-import java.time.ZoneOffset;
 import java.util.Arrays;
-import java.util.Date;
 import java.util.Map;
 
 class JwtClaimsSignerTest {
 
     private static final AlgorithmConstraints PS256_ALGORITHM = new AlgorithmConstraints(
-        AlgorithmConstraints.ConstraintType.WHITELIST, AlgorithmIdentifiers.RSA_PSS_USING_SHA256);
+        AlgorithmConstraints.ConstraintType.PERMIT, AlgorithmIdentifiers.RSA_PSS_USING_SHA256);
```

Comment:
`WHITELIST` is deprecated in favour of `PERMIT` (they both mean the same thing).